### ISSUE TITLE
Allow posts to be merged but not go live until a certain date

### ISSUE THREAD

**Issue by rachelandrew** [GoogleChrome/web.dev#6826]
It would be useful to be able to merge posts that are ready, but not show them live until a "live" date. This would mean that a scheduled release of content doesn't need someone ready to click the button on the day.

**Comment by rachelandrew** [comment]
I'm raising this on web.dev but it would be useful on d.c.c. too.

**Comment by jeffposnick** [comment]
https://github.com/GoogleChrome/webdev-infra/pull/31 is some simple logic that would check the `date` field of an item against any time value in ms (presumably from `Date.now()`). If the date value is greater than the time value, then the item will be marked with `draft: true`.

I _think_ that will give us the behavior we want—items that are marked as draft will be visible in local staging builds, but not visible in public production builds.

How does that sound? Once that other PR is merged, we can add the logic to each site's 11ty config.

**Comment by matthiasrohmer** [comment]
webdev-infra has stubs for this functionality and it is also implemented here:

https://github.com/GoogleChrome/web.dev/blob/main/src/site/_filters/is-live.js#L23-L46

It isn't documented in the handbook though and more granular control (hourly as constrained by the build schedule) would be desired.

**Comment by dstmarthe** [comment]
Closing this as we are moving to a new non-Github based platform. Our new issue tracker lives at https://issuetracker.google.com/issues?q=status:open%20componentid:1400680&s=created_time:desc
